# Useful tools for geneticists

This document is sorted by year and category.


|               Name              | Category |       Description      | Citation
|:-------------------------------:|:--------:|----------------------|:---------:|
|[PLINK v1.9](https://www.cog-genomics.org/plink/1.9/)|Data Manipulation|<ul><li>Genetic data processing</li><li>Quality Control</li><li>Perform linear/logistic regression</li><li>Linkage disequilibrium evalution</li><li>Principal component analysis</li><li>Compatible with other formats (VCF, Oxford)</li></ul>|[https://doi.org/10.1186/s13742-015-0047-8](https://doi.org/10.1186/s13742-015-0047-8)|
|[GATK v3.8](https://gatk.broadinstitute.org/hc/en-us)|Data Manipulation|<ul><li>Variant Calling</li><li>Quality Control</li><li>Filter for depth of coverage</li><li>Filter for genotype quality</li><li>File format: VCF</li></ul>|[http://dx.doi.org/10.1101/gr.107524.110](http://dx.doi.org/10.1101/gr.107524.110)|
|[GCTA](https://cnsgenomics.com/software/gcta/)|Post-GWAS analysis|<ul><li>Estimate the proportion of phenotypic variance explained by all genome-wide SNPs for complex traits</li><li>Estimate the genetic relationships among individuals in GWAS data</li><li>Principal component analysis</li><li>Conditional & joint analysis of GWAS summary statistics without individual-level genotype data</li></ul>|[https://doi.org/10.1016/j.ajhg.2010.11.011](https://doi.org/10.1016/j.ajhg.2010.11.011)|
|[LD Score Regression](https://github.com/bulik/ldsc)|Post-GWAS analysis|<ul><li>Distinguish polygeneity and population strtification</li><li>Estimate heritability and genetic correlation from GWAS summary statistics</li></ul>|[https://doi.org/10.1038/ng.3211](https://doi.org/10.1038/ng.3211)|
|[SKAT](https://cran.r-project.org/web/packages/SKAT/index.html)|Rare variant testing|<ul><li>Perform rare variants testing in the form of optimized burden or kernel tests</li><li>File format: plink bfiles</li></ul>|[https://doi.org/10.1016/j.ajhg.2012.06.007](https://doi.org/10.1016/j.ajhg.2012.06.007)|
