

# Useful tools for geneticists

This document is sorted by year and category.


|               Name              | Category |       Description      | Citation
|:-------------------------------:|:--------:|----------------------|:---------:|
|[PLINK v1.9](https://www.cog-genomics.org/plink/1.9/)|Data Manipulation|<ul><li>Genetic data processing</li><li>Quality Control</li><li>Perform linear/logistic regression</li><li>Linkage disequilibrium evalution</li><li>Principal component analysis</li><li>Compatible with other formats (VCF, Oxford)</li></ul>|[Chang, et al. Second-generation PLINK: rising to the challenge of larger and richer datasets (2015)](https://doi.org/10.1186/s13742-015-0047-8)|
|[GATK](https://gatk.broadinstitute.org/hc/en-us)|Data Manipulation|<ul><li>Variant Calling</li><li>Quality Control</li><li>Filter for depth of coverage</li><li>Filter for genotype quality</li><li>File format: VCF</li></ul>|[McKenna, et al. The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data (2010)](http://dx.doi.org/10.1101/gr.107524.110)|
|[GCTA](https://cnsgenomics.com/software/gcta/)|Post-GWAS analysis|<ul><li>Estimate the proportion of phenotypic variance explained by all genome-wide SNPs for complex traits</li><li>Estimate the genetic relationships among individuals in GWAS data</li><li>Principal component analysis</li><li>Conditional & joint analysis of GWAS summary statistics without individual-level genotype data</li></ul>|[Yang, et al. GCTA: A Tool for Genome-wide Complex Trait Analysis (2010)](https://doi.org/10.1016/j.ajhg.2010.11.011)|
|[LD Score Regression](https://github.com/bulik/ldsc)|Post-GWAS analysis|<ul><li>Distinguish polygeneity and population strtification</li><li>Estimate heritability and genetic correlation from GWAS summary statistics</li></ul>|[Bulik-Sullivan, et al. LD Score regression distinguishes confounding from polygenicity in genome-wide association studies (2015)](https://doi.org/10.1038/ng.3211)|
|[rvtests](http://zhanxw.github.io/rvtests/)|Rare variant testing|<ul><li>Perform multiple type of rare variants testing such as single variant score test, burden test, variable threshold test, SKAT test, fast linear mixed model score test</li><li>Extremely effective for large sample analysis</li><li>File format: vcf, BGEN, plink bfiles</li></ul>|[Zhan, et al. RVTESTS: An Efficient and Comprehensive Tool for Rare Variant Association Analysis Using Sequence Data (2016)](https://doi.org/10.1093/bioinformatics/btw079)||
[SKAT](https://cran.r-project.org/web/packages/SKAT/index.html)|Rare variant testing|<ul><li>Perform rare variants testing in the form of optimized burden or kernel tests</li><li>File format: plink bfiles</li></ul>|[Lee, et al. Optimal Unified Approach for Rare-Variant Association Testing with Application to Small-Sample Case-Control Whole-Exome Sequencing Studies (2012)](https://doi.org/10.1016/j.ajhg.2012.06.007)|
|[MetaSKAT](https://cran.r-project.org/web/packages/MetaSKAT/index.html)|Rare variant testing|<ul><li>Perform meta-analysis of SKAT statistics</li><li>File format: plink bfiles</li></ul>|[Lee, et al. General Framework for Meta-analysis of Rare Variants in Sequencing Association Studies (2013)](https://doi.org/10.1016/j.ajhg.2013.05.010)|
|[SCANG](https://github.com/zilinli1988/SCANG)|Rare variant testing|<ul><li>Perform rare variant testing across the genome</li></ul>|[Li, et al. Dynamic Scan Procedure for Detecting Rare-Variant Association Regions in Whole Genome Sequencing Studies (2019)](https://doi.org/10.1016/j.ajhg.2019.03.002)|
|[STAAR](https://github.com/xihaoli/STAAR)|Rare variant testing|<ul><li>Perform rare variant testing across the genome using annotation information</li><li>Accounts for population structure</li><li>Scalable for WGS study with continuous and dichotomous traits</li></ul>|[Li, et al. Dynamic incorporation of multiple in-silico functional annotations empowers rare variant association analysis of large whole genome sequencing studies at scale (2020)](https://doi.org/10.1038/s41588-020-0676-4)|
|[ACAT](https://github.com/yaowuliu/ACAT)|Rare variant testing|<ul><li>Perform p-value combining for variant-level p-values</li><li>Powerful when small amount of variants are causal</li></ul>|[Liu, et al. ACAT: A Fast and Powerful p Value Combination Method for Rare-Variant Analysis in Sequencing Studies (2019)](https://doi.org/10.1016/j.ajhg.2019.01.002)|
|[TwoSampleMR](https://mrcieu.github.io/TwoSampleMR/index.html)|Causality testing|<ul><li>Mendelian Randomization</li><li>Includes different tests such as MR Egger, MR Steiger</li></ul>|[Hemani, et al. Orienting the causal relationship between imprecisely measured traits using GWAS summary data (2017)](https://doi.org/10.1371/journal.pgen.1007081)|
|[FUSION/TWAS](http://gusevlab.org/projects/fusion/)|Transcriptome-wide association study|<ul><li>Employs imputed gene expression models. Models are constructed from GTEx *cis*-eQTL across different tissues.</li><li>Uses Bayesian sparse linear mixed model (BSLMM) model, a combination of Bayesian variable selection model (BVSR) and linear mixed model (LMM) by assuming a normal mixture prior.)</li><li>Assumes a parametric prior</li></ul>|[Gusev et al. Integrative approaches for large-scale transcriptome-wide association studies (2016)](https://doi.org/10.1038/ng.3506)|
|[MetaXcan/PrediXcan](https://github.com/hakyimlab/MetaXcan)|Transcriptome-wide association study|<ul><li>Employs imputed gene expression models. Models are constructed from GTEx *cis*-eQTL across different tissues.</li><li>Uses Elastic-Net variable selection model (Combination of LASSO and Ridge penalty on *cis*-eQTLs effect size).</li><li>Assumes a parametric prior</li></ul>|[Gamazon, et al. A gene-based association method for mapping traits using reference transcriptome data. (2015)](https://dx.doi.org/10.1038%2Fng.3367)|
|[EpiXcan](https://bitbucket.org/roussoslab/epixcan/)|Transcriptome-wide association study|<ul><li>Perform TWAS including epigonomic data</li></ul>|[Wen, et al. Integrative transcriptome imputation reveals tissue-specific and shared biological mechanisms mediating susceptibility to complex traits. (2019)](https://doi.org/10.1038/s41467-019-11874-7)|
|[TIGAR](https://github.com/yanglab-emory/TIGAR)|Transcriptome-wide association study|<ul><li>Perform TWAS using nonparametric Bayeisan Dirichlet Process Regression</li></ul>|[Nagpal, et al. TIGAR: An Improved Bayesian Tool for Transcriptomic Data Imputation Enhances Gene Mapping of Complex Traits. (2019)](https://doi.org/10.1016/j.ajhg.2019.05.018)|
|[PTWAS](https://github.com/xqwen/ptwas)|Transcriptome-wide association study|<ul><li>Perform TWAS with rigurous causal inference models</li></ul>|[Chen, et al. Investigating tissue-relevant causal molecular mechanisms of complex traits using probabilistic TWAS analysis. (2020)](https://doi.org/10.1371/journal.pgen.1008720)|
|[coloc](https://github.com/chr1swallace/coloc)|Colocalization|<ul><li>Perform genetic colocalisation analysis of two potentially related phenotypes</li></ul>|[Wallace, et al. Eliciting priors and relaxing the single causal variant assumption in colocalisation analyses. (2020)](https://doi.org/10.1371/journal.pgen.1008720)|
|[ENLOC](https://github.com/xqwen/fastenloc)|Colocalization|<ul><li>Utilizes functional data to improve fine-mapping of SNPs</li></ul>|[Wen, et al. Integrating functional data to prioritize causal variants in statistical fine-mapping studies. (2017)](https://doi.org/10.1371/journal.pgen.1004722)|
|[PAINTOR](https://github.com/gkichaev/PAINTOR_V3.0/)|Fine-mapping|<ul><li>Utilizes natural Bayesian hierarchical model</li><li>coloc and eCAVIAR are special cases of the proposed approach</li></ul>|[Kichaev, et al. Integrating Molecular QTL Data into Genome-wide Genetic Association Analysis: Probabilistic Assessment of Enrichment and Colocalization. (2017)](https://doi.org/10.1371/journal.pgen.1006646)|
|[FOCUS](https://github.com/bogdanlab/focus)|Fine-mapping|<ul><li>Fine-map TWAS test statistics</li></ul>|[Mancuso, et al. Probabilistic fine-mapping of transcriptome-wide association studies. (2019)](https://doi.org/10.1038/s41588-019-0367-1)|
